static/app/views/settings/account/notifications/fields2.tsx

```diff
@@ -164,6 +171,24 @@ const CATEGORY_QUOTA_FIELDS = Object.values(DATA_CATEGORY_INFO)
     };
   });
 
+// Define the combined Seer budget field
+const quotaSeerBudgetField = {
+  // This maps to NotificationSettingEnum.QUOTA_SEER_BUDGET
+  name: 'quotaSeerBudget',
+  label: t('Seer Budget'),
+  help: tct(`Receive notifications for your Seer budget. [learnMore:Learn more]`, {
+    learnMore: (
+      <ExternalLink
+        href={getPricingDocsLinkForEventType(DataCategoryExact.SEER_AUTOFIX)}
+      />
+    ),
+  }),
+  choices: [
+    ['always', t('On')],
+    ['never', t('Off')],
+  ] as const,
+};
+
```

Comment:
Future todo: would be nice to derive this from some thing similar to `DATA_CATEGORY_INFO` but for reserved budget categories.